wallet/chain/p/builder/context.go

```diff
@@ -12,21 +12,15 @@ import (
 	"github.com/ava-labs/avalanchego/utils/constants"
 	"github.com/ava-labs/avalanchego/utils/logging"
 	"github.com/ava-labs/avalanchego/vms/avm"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs/fee"
 )
 
 const Alias = "P"
 
 type Context struct {
-	NetworkID                     uint32
-	AVAXAssetID                   ids.ID
-	BaseTxFee                     uint64
-	CreateSubnetTxFee             uint64
-	TransformSubnetTxFee          uint64
-	CreateBlockchainTxFee         uint64
-	AddPrimaryNetworkValidatorFee uint64
-	AddPrimaryNetworkDelegatorFee uint64
-	AddSubnetValidatorFee         uint64
-	AddSubnetDelegatorFee         uint64
+	NetworkID       uint32
+	AVAXAssetID     ids.ID
+	StaticFeeConfig fee.StaticConfig
```

Comment:
Now that the config doesn't have the `CreateAssetFee` - we can just use the config here directly 🥳